src/bp-templates/bp-nouveau/includes/activity/widgets.php

```diff
@@ -120,6 +120,7 @@ public function widget( $args, $instance ) {
 			'action'       => join( ',', $type ),
 			'primary_id'   => 0,
 			'secondary_id' => 0,
+			'privacy' 	   => apply_filters( 'bb_activity_privacy_values', $privacy ),
```

Comment:
@ngolwalkar No need for this as well just pass the `'scope'        => 'all',` to `'scope'        => bp_activity_default_scope( 'all' ),`